pyproject.toml

```diff
@@ -33,7 +33,7 @@ dependencies = [
     "gevent ~=21.12.0",
     "influxdb ~= 5.3.1",
     "locust ~= 2.12.2",
-    "locust-influxdb-listener ~= 0.0.9",
+    "locust-influxdb-listener ~= 0.0.10",
```

Comment:
Hello @jmfiola, are you sure the version exists?
I can see the 0.0.9 is the latest version and this line is the only reason why the pre-merge unit tests are failing :) 
https://pypi.org/project/locust-influxdb-listener/#history